js/Source/Request/Request.PollForUpdate.js

```diff
@@ -63,10 +63,10 @@ Request.PollForUpdate = new Class({
     var data = this.request.response.json;
     // if the server provides an 'updated_at' timestamp,
     // store as an attribute
-    if (data.status == 'update'){
+    if (typeof data !== 'undefined' && data.status == 'update'){
```

Comment:
i think the reason i thought that is because if you just run this in the console:
`if(data && data.status) {console.log('hello')}`
you get
`VM43399:2 Uncaught ReferenceError: data is not defined(…)`